client/src/pages/Info/index.tsx

```diff
@@ -1,10 +1,101 @@
 import SecondaryPagesLayout from 'components/layouts/SecondaryPagesLayout';
-import React from 'react';
+import React, { FC, useRef } from 'react';
+import { CgDarkMode } from 'react-icons/cg';
+import { MdWbSunny } from 'react-icons/md';
+import { FaTemperatureLow } from 'react-icons/fa';
+import { getBannerImage } from 'utils/getImages';
+import styles from './styles.module.scss';
+
+interface Ref {
+  current: HTMLDivElement | null;
+}
+
+const Info: FC = () => {
+  const refSectionCurrent = useRef<HTMLInputElement | null>(null);
+  const refForecast = useRef<null | HTMLDivElement>(null);
+  const refTheme = useRef<null | HTMLDivElement>(null);
+
+  const handleClick = (ref: Ref) => {
+    ref.current?.scrollIntoView({ behavior: 'smooth' });
+  };
```

Comment:
please try css solution first with anchors and smooth scrolling
https://developer.mozilla.org/en-US/docs/Web/CSS/scroll-behavior

Reply:
I have tried, but it is not working

Reply:
it should. you can ask Vilius for help (he used this solution) or ask me

Reply:
oh, it's working, but only after second click. That's why I haven't noticed before.

Reply:
it's not working correctly because you wrapped only small part of page with scroll-behavior property, but it should be placed in container that wraps both links and content